opm/simulators/wells/PerforationData.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+  Copyright 2019 SINTEF Digital, Mathematics and Cybernetics.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_PERFORATIONDATA_HEADER_INCLUDED
+#define OPM_PERFORATIONDATA_HEADER_INCLUDED
+
+namespace Opm
+{
+
+/// Static data associated with a well perforation.
+struct PerforationData
+{
+    int cell_index;
+    double connection_transmissibility_factor;
+    int satnum_id;
+};
```

Comment:
Do you envision adding dynamic data to PerforationData at in a later PR? For instance the intensiveQuantity related to that perforation.

Reply:
No. This is for well-static data only. Anything state-related should not go here. Essentially, this is "what is missing in the Connection class", especially the cell index.